CefSharp.Wpf/ChromiumWebBrowser.cs

```diff
@@ -626,8 +626,13 @@ private IntPtr SourceHook(IntPtr hWnd, int message, IntPtr wParam, IntPtr lParam
                         return IntPtr.Zero;
                     }
 
-                    handled = managedCefBrowserAdapter.SendKeyEvent(message, wParam.ToInt32(), lParam);
-                    
+                    const CefEventFlags modifiers = 0;
+
+                    if (managedCefBrowserAdapter.SendKeyEvent(message, wParam.ToInt32(), (IntPtr) AddKeyboardModifiers(modifiers)))
```

Comment:
As `lparam` is representing the native key code I think this is likely to break the keyboard handling.

https://github.com/cefsharp/CefSharp/blob/master/CefSharp.Core/ManagedCefBrowserAdapter.h#L191

Is the Modifier required in the `SourceHook`? Thought it would have been enough to just pass it though on mouse click?